UI/window-basic-status-bar.cpp

```diff
@@ -195,7 +195,11 @@ void OBSBasicStatusBar::UpdateBandwidth()
 	if (++seconds < bitrateUpdateSeconds)
 		return;
 
-	uint64_t bytesSent = obs_output_get_total_bytes(streamOutput);
+	OBSOutputAutoRelease output = obs_weak_output_get_output(streamOutput);
```

Comment:
I would suggest to use OBSGetStrongRef to be consistent with how other places in the UI get the strong reference of a weak pointer.

Reply:
And do this for all of the PR.

Reply:
For everywhere else in the codebase that uses `OBSGetStrongRef`, the type of the receiving variable is one of:
* auto
* OBSSource
* OBSObject

If we were to use `OBSGetStrongRef` here, then it seems the type for `output` should either be `auto` or `OBSOutput`. That said, I'm not sure of the semantic difference off-hand and would have to look into it. I'm not sure if one approach is preferred over the other.

Reply:
Changed to use `OBSGetStrongRef` and `OBSOutput` accordingly.